pkgs/by-name/cl/clevis/package.nix

```diff
@@ -60,6 +60,10 @@ stdenv.mkDerivation (finalAttrs: {
     "man"
   ];
 
+  # TODO: investigate how to prepare the dependencies (and study the dark
+  # corners of cross-compilation in Nixpkgs)
```

Comment:
Just out of curiosity, did you try to cross compile clevis? Could it be that this is what is required for the tests to pass?

Reply:
I tried pkgsMusl it, without success.

Nonetheless the build fails even in native mode, because the various dependencies are not in their proper places.